src/dynamo0p3.py

```diff
@@ -2949,7 +2960,18 @@ def generate(self):
         # Add boundary dofs array to the operator boundary condition
         # kernel (enforce_operator_bc_kernel) arguments
         if self._kern.name.lower() == "enforce_operator_bc_code":
-            self.operator_bcs_kernel(fs_to)
+            # Minimal sanity check - this kernel should only have a single LMA
+            # operator as argument so the first one must be a gh_operator.
+            # Full sanity checking is done in the operator_bcs_kernel()
```

Comment:
I've commented on this below, but there are multiple implementations of the operator_bcs_kernel() in the different subclasses so there is no single implementation. I think the sanity checking should all be done here (in the base class).